pkg/operator/ceph/cluster/mon/health_test.go

```diff
@@ -341,8 +341,8 @@ func TestCheckHealthNotFound(t *testing.T) {
 	// recheck that the "not found" mon has been replaced with a new one
 	cm, err = c.context.Clientset.CoreV1().ConfigMaps(c.Namespace).Get(ctx, EndpointConfigMapName, metav1.GetOptions{})
 	assert.Nil(t, err)
-	if cm.Data[EndpointDataKey] != "a=1.2.3.1:6789,f=:6789" {
-		assert.Equal(t, "f=:6789,a=1.2.3.1:6789", cm.Data[EndpointDataKey])
+	if cm.Data[EndpointDataKey] != "a=1.2.3.1:3300,f=:6789" {
+		assert.Equal(t, "f=:6789,a=1.2.3.1:3300", cm.Data[EndpointDataKey])
```

Comment:
Just making sure that `f` not having an IP and isn't using the `3300` port doesn't mean that something has gone wrong here. I assume it's good, but I am scratching my head a bit wondering what this would mean in the real world.

Reply:
This is an old confusing test... There are really two checks built into this approach.
1. The format for an existing mon remains untouched even during a mon failover when a new mon is added to the list
2. The mons are not ordered, so the check for both possible orders is a bit duplicated and braindead.